packages/python/plotly/requires-optional.txt

```diff
@@ -25,10 +25,12 @@ mock==2.0.0
 pytest==8.1.1
 xarray
 pytz
-
-## orca dependencies ##
-requests
-psutil
+polars
```

Comment:
This isn't exactly a problem with your PR, but I think it would be helpful to clarify the difference between this file (packages/python/plotly/requires-optional.txt) and the files in packages/python/plotly/test_requirements. I think it could be good to consolidate them or recommend that developers use the test requirements file that corresponds to their python version.

Reply:
:+1: thank you - I've moved content from requires-optional.txt to test_requirements/requirements_311_optional.txt. I'll see what I need to do with Python 3.12 as well.